docs/use_case/post_demo_app_intro.md

```diff
@@ -0,0 +1,53 @@
+In this tutorial we are going to present 3 versions of ONEm's #post application. The apps are written using the Django framework ONEm's Python SDK.
+
```

Comment:
It would be good to have a description of #post micro-app in term of what it does and then split the features among versions:
"#post micro-app is a collaborative application that serves as a virtual post board where users can post small adds sharing information or services. Users can post, edit and view adds, search by keyword, contact the add owner, comment and rate. "

Reply:
There's a short description of the features of each micro-app version of #post in the `Introduction` part:
```
    postv1 - a stripped down version of the service; we will be able to create, store and view posts
    postv2 - on top of v1, we will now add a Search wizard so we can search by keyword/code in all saved posts (ours or others)
    postv3 - on top of v2, we will enable interactions with ONEm's restdAPI
```
But you're right - no general description on what #post is and does, so I'll add that.